src/components/datepicker/dateLocaleProvider.js

```diff
@@ -158,6 +158,10 @@
         if (!date) {
           return '';
         }
+				
+        if (typeof date === 'string') {
```

Comment:
The entire point of this function is that it turns a Date object into a string. Under what circumstance would it be passed a string?